webui/src/components/help.jsx

```diff
@@ -1,70 +1,39 @@
 import React from 'react/lib/ReactWithAddons';
 import { Link } from 'react-router';
-
+import {Store} from '../data/store';
```

Comment:
Is 'Store' being used in this file?

Reply:
Not anymore, was used in an earlier commit, thus can be removed.

```suggestion

```

Reply:
Commits are updated